TinyOAuth1/TinyOAuth.cs

```diff
@@ -267,30 +269,19 @@ public async Task<AccessTokenInfo> GetAccessTokenAsync(string requestToken, stri
 
 			if (!string.IsNullOrEmpty(responseText))
 			{
-				string oauthToken = null;
-				string oauthTokenSecret = null;
-				var keyValPairs = responseText.Split('&');
+			    var keyValPairs = HttpUtility.ParseQueryString(responseText);
 
-				for (var i = 0; i < keyValPairs.Length; i++)
-				{
-					var splits = keyValPairs[i].Split('=');
-					switch (splits[0])
-					{
-						case "oauth_token":
-							oauthToken = splits[1];
-							break;
-						case "oauth_token_secret":
-							oauthTokenSecret = splits[1];
-							break;
-					}
-				}
+			    var oauthToken = keyValPairs["oauth_token"];
```

Comment:
Does this throw if the key does not exist? The old behavior would simply keep `keyValPairs` and `oauthToken` null. Maybe we do not want this new behavior?

Reply:
Fair. It would throw I believe. I did this very quickly last night. Wasn't expecting you to get your eyes on it so quickly 😝